src/controllers/handlers.ts

```diff
@@ -148,6 +148,16 @@ export const estateMapPngRequestHandler = async (context: {
     const selected = Object.values(tiles).filter(
       (tile) => tile.estateId && tile.estateId === estateId
     )
+    if (selected.length === 0) {
+      const  headers: Record<string, string> = {
```

Comment:
It has a double whitespace here.
```suggestion
      const headers: Record<string, string> = {
```